src/components/Nav/index.js

```diff
@@ -0,0 +1,17 @@
+import "./nav.css"
+
+function Nav() {
+  return (
+    <nav className="navBar">
+      <a href='/'>Books Swap</a>
+      <div className="right-links">
+      <a href='/'>Available books</a>
+      <a href='/claimed'>Claimed books</a>
+      <a href='#'>Add book</a>
```

Comment:
Don't use a tags for links to other routes as it refreshes the page. Instead you should use either Link or NavLink